core/templates/components/forms/custom-forms-directives/thumbnail-uploader.directive.html

```diff
@@ -12,26 +12,28 @@
       <oppia-thumbnail-display ng-show="!thumbnailIsLoading"
                                [img-src]="editableThumbnailDataUrl"
                                [classes]="['img-thumbnail', 'protractor-test-custom-photo']"
-                               [background]="getBgColor()">
+                               [background]="getBgColor()"
+                               [aspect-ratio]="'4:3'">
       </oppia-thumbnail-display>
       <oppia-thumbnail-display ng-if="useLocalStorage && newThumbnailDataUrl"
                                [img-src]="newThumbnailDataUrl"
                                [classes]="['img-thumbnail', 'protractor-test-custom-photo']"
-                               [background]="localStorageBgcolor">
+                               [background]="localStorageBgcolor"
+                               [aspect-ratio]="'4:3'">
```

Comment:
Hi @lkbhitesh07, looks like you are still hard-coding the aspect ratio here. Should we be passing  getAspectRatio() instead?

https://github.com/oppia/oppia/blob/e2895b00266e9c12262c92ac99572fe40f0816bc/core/templates/components/forms/custom-forms-directives/thumbnail-uploader.directive.ts#L42